libc/config/linux/x86_64/entrypoints.txt

```diff
@@ -494,6 +494,10 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.round
     libc.src.math.roundf
     libc.src.math.roundl
+    libc.src.math.roundeven
+    libc.src.math.roundevenf
+    libc.src.math.roundevenl
+    libc.src.math.roundevenf128
```

Comment:
Move `f128` variant down with other `f128` functions where they are guarded.

Reply:
Sorry! I somehow didn't notice that. Fixed now.